backend/copr_backend/sign.py

```diff
@@ -116,6 +117,8 @@ def gpg_hashtype_for_chroot(chroot, opts):
         # epel-8+ and not _all_ the package data on backend.
         if parts[0] in el_chroots:
             el_version = version_part
+            if el_version in ["rawhide"]:
+                return "sha1"
```

Comment:
This method is likely wrong.  While working on the code, I thought sha256 signatures are not supported on EL6, but they are.  I think that if rhel-rawhide was re-created, we should mark it sha256.

Reply:
Okay, I will fix the test case then.